service/awsconfig/v7/templates/cloudformation/guest/outputs.go

```diff
@@ -8,6 +8,8 @@ Outputs:
     Value: {{ .Outputs.Master.InstanceType }}
   MasterCloudConfigVersion:
     Value: {{ .Outputs.Master.CloudConfig.Version }}
+  {{ .Outputs.Worker.ASG.Key }}:
+    Value: !Ref {{ .Outputs.Worker.ASG.Tag }}
```

Comment:
We need the ASG name, like the real name in order to work with the lifecycle hooks later on.